### PR TITLE
fix `openid` scope issue as client is requesting for `id_token`

### PR DESCRIPTION
**Pull request by KhizerRehan**
This PR fix the `infinite` loop issue due to missing of providing scope of `openid` As client request is asking for the `response_type=id_token`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,26 +27,39 @@ In order to run this example you need to setup some configurations in Curity ser
...
 
 4. This client (accessible from `Token Service -> Clients -> client-assisted-example -> Edit Client`) should have `Implicit Flow` and `Assisted Token` capabilities selected under the `Capabilities` section.
 
+
    ![image](./docs/images/client-capabilities.png)
 
-5. Update the `Redirect URIs` and `Allowed Origins` settings for the `client-assisted-example` Client. The redirect URI should have `http://localhost:4200`. This is the URL where the Angular CLI will be hosting the SPA. The allowed origin should be the same or, for testing purposes, you can also use `*`.
+
+5. Navigate to `OAuth/OpenID Settings` section make sure to add 
+`opendid` scope.
```
Comment (travisspencer) [inline README.md:52]: typo -- `openid` -- extra D there.

**Review by travisspencer — CHANGES_REQUESTED** [review]

**Review by travisspencer — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -27,26 +27,39 @@ In order to run this example you need to setup some configurations in Curity ser
...
+![image](docs/images/openid-scope.png)
+
+
+6. Update the `Redirect URIs` and `Allowed Origins` settings for the `client-assisted-example` Client. The redirect URI should have `http://localhost:4200`. This is the URL where the Angular CLI will be hosting the SPA. The allowed origin should be the same or, for testing purposes, you can also use `*`.
+   
 
    ![image](./docs/images/client-application-settings.png)
 
-6. Commit the changes, and you are all setup!
+7. `Validate` and `Commit` the changes and you are all setup.
```
Comment (Waseem826) [inline README.md:62]: No need to say `Validate` because `Commit` action also validates.

File: README.md [hunk 3]
```diff
@@ -27,26 +27,39 @@ In order to run this example you need to setup some configurations in Curity ser
...
 
 4. This client (accessible from `Token Service -> Clients -> client-assisted-example -> Edit Client`) should have `Implicit Flow` and `Assisted Token` capabilities selected under the `Capabilities` section.
 
+
    ![image](./docs/images/client-capabilities.png)
 
-5. Update the `Redirect URIs` and `Allowed Origins` settings for the `client-assisted-example` Client. The redirect URI should have `http://localhost:4200`. This is the URL where the Angular CLI will be hosting the SPA. The allowed origin should be the same or, for testing purposes, you can also use `*`.
+
+5. Navigate to `OAuth/OpenID Settings` section make sure to add 
+`openid` scope.
```
Comment (Waseem826) [inline README.md:52]: Move this to previous line.

**Review by Waseem826 — COMMENTED** [review]
Screenshot for openid scope should be consistent with other screenshots (i.e. no top nav bar and full width page).

**Review by Waseem826 — APPROVED** [review]

**Pull request merged**